app/graphql/mutations/term/base_mutation.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Mutations::Term::BaseMutation < Mutations::BaseMutation
+  def find_vocabulary!(string_key)
```

Comment:
Can we rename this to something like `find_unlocked_vocabulary!` ?  Given that the current name looks like a plain object finder method, I feel like it's a little unintuitive that it would raise an error if the vocabulary is locked.  Name change would help.

Reply:
I'll make that change.

Reply:
Thanks!